bindings/python/native/src/lib.rs

```diff
@@ -48,10 +55,346 @@ pub fn send_message(handle: &ClientMessageHandler, message_type: String) -> Resu
     Ok(serde_json::to_string(&response)?)
 }
 
+#[pyfunction]
+#[allow(clippy::too_many_arguments)]
+/// Create OutputDto::Treasury JSON string
+pub fn create_treasury_output(amount: u64) -> Result<String> {
```

Comment:
Here it's also useless

Reply:
To make it consistent so I add it :)

Reply:
I removed it at [31ee5c1](https://github.com/iotaledger/iota.rs/pull/943/commits/31ee5c11fd25b70e0bf75f94634592f68da67ace)